src/shared/clock-util.c

```diff
@@ -138,11 +138,14 @@ int clock_reset_timewarp(void) {
 }
 
 #define EPOCH_FILE "/usr/lib/clock-epoch"
+#define DELTA_THRESHOLD (USEC_PER_YEAR*15)
```

Comment:
we generally suffix defines that refer to time values with `_USEC` to make the unit clear